### DISCUSSION TITLE
Welcome to rust_magazine_2021 Discussions!

### DISCUSSION THREAD

**Discussion by ZhangHanDong** [RustMagazine/rust_magazine_2021#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZhangHanDong** [comment]
大家对 RustMagazine 中文月刊 有什么建议或意见都可以提。

**Reply by iamazy** [reply]
汉东老师，如何参与编辑小组呢

**Reply by ZhangHanDong** [reply]
请问你在微信群吗

**Reply by iamazy** [reply]
不在

**Reply by ZhangHanDong** [reply]
那哪里可以找到你呢

**Reply by iamazy** [reply]
在微信中可以通过qq号搜到我：1448588084

**Reply by ZhangHanDong** [reply]
ok
